managed/devops/bin/yb_platform_util.py

```diff
@@ -0,0 +1,698 @@
+        universe_config_json['currentClusterType'] = 'PRIMARY'
+        # Call universe config api to get the update config data.
+        universe_config_json = self.__post_universe_config(universe_config_json)
+        if universe_config_json:
+            task_id = self.__create_universe_from_config(universe_config_json)
+            print('Universe create requested successfully. Use {0} as task id to get status of universe.'.format(task_id))
+            if not self.args.no_wait:
+                self.__task_progress_bar(task_id)
+        else:
+            sys.stderr.write('Unable to Create Universe for Customer {0}'.format(customer_uuid))
+
+
+    def get_single_customer_uuid(self):
+        """
+        Get customer UUID. Raise error in case multiple customer is present. 
+
+        :return: None
+        """
+        if self.args.customer_uuid:
+            self.customer_uuid = self.args.customer_uuid
+        else:
+            customer_url = '{0}/api/v1/customers'.format(self.base_url)
+            data = self.__call_api(customer_url)
+            if (len(data) == 1):
+                self.customer_uuid = str(data[0])
+            else:
+                sys.stderr.write('Muliple customer UUID present, Please provide customer UUID')
+                exit(1)
+
+
+    def get_regions_data(self):
+        """
+        Delete the universe by providing UUID.
+
+        :return: All available regions
+        """
+        provider_url = '{0}/api/v1/customers/{1}/regions'.format(self.base_url, self.customer_uuid)
+        region_data = self.__call_api(provider_url)
+        regions = []
+        for each_region in region_data:
+            zones = []
+            for each_zone in each_region.get('zones'):
+                zones.append({
+                    'zone_name': each_zone.get('name'),
+                    'zone_uuid': each_zone.get('uuid')
+                })
+            regions.append({
+                'zones': zones,
+                'region_name': each_region.get('name'),
+                'region_uuid': each_region.get('uuid'),
+                'provider': each_region.get('provider').get('code')
+            })
+        print(json.dumps(regions))
+    
+
+    def get_provider_data(self):
+        """
+        Delete the universe by providing UUID.
+
+        :return: All available providers
+        """
+        provider_url = '{0}/api/v1/customers/{1}/providers'.format(self.base_url, self.customer_uuid)
+        provider_data = self.__call_api(provider_url)
+        providers = []
+        for each_provider in provider_data:
+            providers.append({
+                'name': each_provider.get('name'),
+                'uuid' : each_provider.get('uuid')
+            })
+        print(json.dumps(providers))
+
+
+    def delete_universe(self):
+        """
+        Function to do delete action.
+
+        :return: None
+        """
+        confirmation = self.args.yes and 'y'
```

Comment:
Could you please explain the purpose of `and 'y'` ?

Reply:
We are taking confirmation from users to delete universe and they will enter y/yes. 
Now they is one more way where user can pass --yes while running the command. 
In that case, I am checking if the user has used --yes then I am adding 'y'(same as user input).  
self.args.yes will have true or false value. true means user has passed --yes and we don't require to take confirmation from user again. So i added user input as 'y'
.

Reply:
I don't get it. If the user enter yes as the argument, we can still do the next check, right? Why is the `and y` important? Also, you are changing confirmation from boolean to string. Bad practice. 
Check confirmation, if true, go ahead. If not true, get input, and then update confirmation to true or false based on that.